pages/_app.js

```diff
@@ -109,6 +110,8 @@ function MyApp({ Component, pageProps }) {
         onDeleteWorkout={handleDeleteWorkout}
         onAddExercise={handleAddExerciseToWorkOut}
         onDeleteExercise={handleDeleteExercise}
+        exerciseAdded={exerciseAdded}
+        setExerciseAdded={setExerciseAdded}
```

Comment:
```suggestion
  updateAddedExercises={updateAddedExercises}
```
(die Setterfunktion würdest du nicht runtergeben)